File: Capta.Domain/Jogador.cs
using System.Collections.Generic;

namespace Capta.Domain
{
    public class Jogador
    {
        public int JogadorId { get; set; }
        public string Nome { get; set; }
        public decimal Peso { get; set; }
        public decimal Altura { get; set; }
        public IEnumerable<JogadorHabilidade> JogadorHabilidades { get; set; }
        public int? TimeId { get; set; }
        public Time Time { get; }
    }
}

File: CaptaApp/src/app/_models/Jogador.ts
import { Habilidade } from './Habilidade';
import { Time } from '@angular/common';

export class Jogador {
  jogadorId: number;
  nome: string;
  peso: number;
  altura: number;
  jogadorHabilidades: Habilidade[];
  time: Time;
}
